features/distributed/coordination/api/src/main/java/org/opennms/features/distributed/coordination/api/Role.java

```diff
@@ -0,0 +1,38 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2018 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2018 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.features.distributed.coordination.api;
+
+/**
+ * The valid roles when participating in a redundant cluster.
+ */
+public enum Role {
+    ACTIVE,
+    STANDBY,
+    UNKNOWN
```

Comment:
Can we define UNKNOWN first? Otherwise Active may be the default which may not be the intention.

Reply:
Sure I can change it, I just had them in alphabetical order. What do you mean by the default? As far as I know this isn't being used in such a way that it could have a default value.